app/views/lti_deployment/_line_items.html.erb

```diff
@@ -0,0 +1,24 @@
+<table>
+  <thead>
+  <tr>
+    <th><%= t("lti.lti_settings") %></th>
+  </tr>
+  </thead>
+  <tbody>
+  <%= fields_for :lti_deployment do |f|  %>
+  <% @lti_deployments.each do |deployment| %>
+    <tr>
+      <td>
+
+        <%= f.label deployment.id, "#{deployment.lms_course_name} on #{URI(deployment.lti_client.host).host}", value: deployment.id %>
```

Comment:
1. The label string should be internationalized. ("on" is only English)
2. Is the `value` argument necessary? I'm not sure what it does for a `label`